apps/prairielearn/src/ee/lib/ai-grading/ai-grading-stats.ts

```diff
@@ -65,10 +81,13 @@ export async function fillInstanceQuestionColumns<
     return acc;
   }, {});
 
-  const results: WithAIGradingStats<T>[] = [];
+  const results: FillInstanceQuestionColumnEntriesResultType<T>[] = [];
 
-  for (const base_instance_question of instance_questions) {
-    const instance_question: WithAIGradingStats<T> = {
+  for (const item of info_with_instance_questions) {
+    const base_instance_question = item.instance_question;
+    const { instance_question: _unused, ...other_info } = item;
```

Comment:
Is this destructuring really needed? You can just spread all of `item` in on the `results.push` below, the `instance_question` property is overwritten after the spread.